ktor-server/ktor-server-plugins/ktor-server-content-negotiation/jvmAndNix/src/io/ktor/server/plugins/contentnegotiation/RequestConverter.kt

```diff
@@ -5,20 +5,27 @@
 package io.ktor.server.plugins.contentnegotiation
 
 import io.ktor.http.*
-import io.ktor.serialization.*
+import io.ktor.http.content.*
 import io.ktor.server.application.*
 import io.ktor.server.plugins.*
 import io.ktor.server.request.*
-import io.ktor.util.*
+import io.ktor.util.logging.*
+import io.ktor.util.reflect.*
 import io.ktor.utils.io.*
 import io.ktor.utils.io.charsets.*
 
 internal fun PluginBuilder<ContentNegotiationConfig>.convertRequestBody() {
+    val log = application.log
```

Comment:
why use application log and not create new instance as in other plugins?

Reply:
fixed